fronius: extend if battery storage is involved (#21072)
home-assistant/home-assistant.github.io: source/_integrations/fronius.markdown

- For `Grid consumption` use the meters `Energy real consumed` entity.
- For `Return to grid` use the meters `Energy real produced` entity.
- For `Solar production`: 
  - If no battery is involved at an inverter: Add each inverters `Energy total` entity.

>   - If a battery is involved at an inverter: Use [Riemann sum](/integrations/integration/) over `Power photovoltaics` entity (from power_flow endpoint found in your `SolarNet` device)

Reviewer: So `Energy Total` sums PV and Battery (-)charge, discharge energy?
Oh dear, this API is a mess...

Author: I also understand it from an API perspective, „Energy total“ on the inverter is whatever is converted from DC to AC. Independent of whether its coming from a solar panel or a battery storage.

Reviewer: ```suggestion
- For `Solar production`: 
  - If no battery is connected to an inverter: Add each inverters `Energy total` entity.
  - If a battery is connected to an inverter: Use [Riemann sum](/integrations/integration/) over `Power photovoltaics` entity (from power_flow endpoint found in your `SolarNet` device)
```
I'm not particular fluent in english - is "involved at" the correct term?

Author: Indeed, thank you, updated
